File: hw/unda/U3/OOPSingleton/src/ec/edu/espe/view/Calculator.java
package ec.edu.espe.view;
import ec.edu.espe.model.USTax;

public class Calculator {
    
    public static void main(String[] args) {
        
        USTax tax = USTax.getInstance();

        float basePrice = 20f;
        float total = tax.salesTotal(basePrice);

        System.out.println("Final price = $" + total);
    }
}

File: hw/paillacho/U3/HW18Singleton/CalculatorPython/view/Calculator.py
# ec/edu/espe/view/calculator.py
# Author: Carlos Paillacho

from ec.edu.espe.model.us_tax import USTax


def main():
    tax = USTax.get_instance()

    base_price = 100.0
    total = tax.sales_total(base_price)

    print(f"Final price: ${total:.2f}")


if __name__ == "__main__":
    main()
